other-docs/guides/launching-a-site-on-altis.md

```diff
@@ -0,0 +1,181 @@
+- [ ] Spin up environment
+- [ ] Deploy to environment
+- [ ] Dump original database and import new environment
+- [ ] Perform search/replace where necessary
+- [ ] Confirm site is operating normally
+
+### domain-staging
+
+- [ ] Create terraform config in humanmade/infrastructure-config/terraform/apps
+- [ ] Retrieve the latest AMI for the ECS EC2 Cluster, add to TF config
+- [ ] Gather all the domains the stack will use to serve content from, add to TF config
+- [ ] Create a valid SSL Certificate in ACM, add to TF Config
+- [ ] Update GitHub repository details in TF config
+- [ ] Retrieve migration parameters using script, add to TF config
+- [ ] Add DNS Terraform config
+- [ ] Spin up environment
+- [ ] Deploy to environment
+- [ ] Dump original database and import new environment
+- [ ] Perform search/replace where necessary
+- [ ] Confirm site is operating normally
+
+### domain-production
+
+- [ ] Create terraform config in humanmade/infrastructure-config/terraform/apps
+- [ ] Retrieve the latest AMI for the ECS EC2 Cluster, add to TF config
+- [ ] Gather all the domains the stack will use to serve content from, add to TF config
+- [ ] Create a valid SSL Certificate in ACM, add to TF Config
+- [ ] Update GitHub repository details in TF config
+- [ ] Retrieve migration parameters using script, add to TF config
+- [ ] Add DNS Terraform config
+- [ ] Spin up environment in read-only migration mode
+- [ ] Deploy to environment
+- [ ] Disconnect from source environment
+- [ ] Update MySQL Password Need to establish a better process around this
+- [ ] Perform search/replace where necessary
+- [ ] Confirm site is operating normally
+
+### Production Data Migration
+Reference : https://github.com/humanmade/terraform-app-stack/blob/master/MIGRATING.md#migration-checklist
+- [ ] Communicate an edit lock with the client.
+- [ ] Spin up the Terraform stack.
+- [ ] Verify the stack is functioning correctly
+- [ ] Point Terraform to the original CloudFront distribution
+- [ ] Remove the existing distribution from Terraform's state
+- [ ] Import the original distribution
+- [ ] Update CloudFront domains with the values from the original distribution
+- [ ] Plan the changes:
+- [ ] Verify the changes looks
+- [ ] Inform client edit lock has begun.
+- [ ] Prepare to flip the switch
+- [ ] Flip the switch:
+- [ ] Monitor the RequestCount metric on the original Load Balancer
+- [ ] Disable Cavalcade on the CloudFormation stack
+- [ ] Verify the new RDS Cluster is in sync with the Original RDS Instance
+- [ ] Disable Replication
+- [ ] Enable writes on the new infrastructure
+- [ ] Delete the dangling CloudFront distribution.
+- [ ] Update the CloudFormation stack template to retain CloudFront distribution.
+- [ ] Delete Cloudformation stack once after the client QA.
+
+```
+
+## Post-launch
+
+After the site launch is completed, there may be some items left to clean up or check.
+
+- Do you need to set the default branch back to `main` in your repository?
```

Comment:
Might be related to the migration branch note above? Not sure this is necessary

Reply:
That's probably accurate. New sites wouldn't likely need to switch deploy branches.